organizers/2018-06-25.html

```diff
@@ -0,0 +1,140 @@
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>June 25th 2018</h2>
+    <img data-src="/img/django-logo-negative.png" style="width:40%" alt="Django logo" class="plain"></img>
+</section>
+
+<section>
+    <h1>Hello, I'm Marco!</h1>
+    <h2>👋😆</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Thread</strong></li>
+        <li><strong>Time Out</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain"></img>
+    <p>
+        <em>“Festival package travel startup. 50+ staff, high pay, ping pong + beer.”</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/thread.png" style="width:10%" alt="Thread" class="plain"></img>
+    <p>
+        <em>We help people to feel happier and more self-confident, by making it easy to dress well, using stylists and data science. Dedicated to maintaining a great working environment, minimising politics and maximising impact.</em>
+    </p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain"></img>
+    <p>
+        <em>The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.</em>
+    </p>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>🎺 Django News</h2>
+    <h2>🎺 Python News</h2>
+    <h2>🎺 Developers News</h2>
+</section>
+
+<section style="text-align: left">
+    <h1 style="text-align: center;">📋 Today's Schedule</h1>
+    <h2 style="text-align: center;">Talks</h2>
+    <ul>
+        <li>
+            Matthew Hall - 10 Lessons From 10 Years of Selling Dev Work
+        </li>
+        <li>
+            Lili Kastilio - pip install disaster-waiting-to-happen
+        </li>
+    </ul>
+</section>
+
+<section style="text-align: left">
+    <h1>📋 Today's Schedule</h1>
+    <h2>Eating and Drinking</h2>
+    <ol>
+        <li><strong>Until ~20:45</strong> - Drinks and Pizza at<br>The Space Bar</li>
+        <li><strong>20:45 - close</strong>: Red Lion Pub (opposite)</li>
+    </ol>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>Bugfix release</h2>
+    <ul>
+        <li>2.0.6 bugfix release</li>
+        <li>2.1 beta release</li>
+	    <li><a href="https://www.djangoproject.com/weblog/">https://www.djangoproject.com/weblog/</a></li>
+    </ul>
+</section>
+
+<section>
+    <h1>🎺 Django News</h1>
+    <h2>DjangoCon Europe 2018</h2>
+    <ul>
+        <li>May 23-27, 2018 · Heidelberg, Germany</li>
```

Comment:
May?

Reply:
Yes is past, I added a mention because was after the last meetup and they released the videos.

https://2018.djangocon.eu/